packages/node_modules/@node-red/editor-client/src/types/node-red/func.d.ts

```diff
@@ -27,56 +27,41 @@ interface NodeStatus {
     /** The shape property can be: ring or dot */
     shape?: string,
     /** The text to display */
-    text?: string
+    text?: string|boolean|number
 }
 
 declare class node {
     /** 
     * Send 1 or more messages asynchronously 
     * @param {object | object[]} msg  The msg object 
     * @param {Boolean} [clone=true]  Flag to indicate the `msg` should be cloned. Default = `true`   
-    * @example Send 1 msg to output 1 
```

Comment:
Monaco having problems rendering the example in the intellisense info tip - removed the `@example`